LICENSE

```diff
@@ -1,22 +1,22 @@
-The MIT License (MIT)
```

Comment:
Somewhere in the process your Git client or your editor decided to mess with the line endings here :) (Same with some other files) Anyway, it's not really a big deal.